be/src/util/buffered_stream.h

```diff
@@ -14,7 +14,9 @@ class RandomAccessFile;
 
 class IBufferedInputStream {
 public:
-    virtual Status get_bytes(const uint8_t** buffer, size_t offset, size_t* nbytes) = 0;
+    virtual Status get_bytes(const uint8_t** buffer, size_t offset, size_t* nbytes, bool peek = false) = 0;
```

Comment:
For virtual function better not to use default value. https://google.github.io/styleguide/cppguide.html#Default_Arguments

Reply:
> For virtual function better not to use default value. https://google.github.io/styleguide/cppguide.html#Default_Arguments

OK